Tour/subscripts.md

```diff
@@ -23,7 +23,7 @@ To better understand, let us instrument the subscript to observe its behavior. S
 ```
 subscript min(_ x: Int, _ y: Int): Int {
   print("enter")
-  yield if y > x { y } else { x }
+  yield if y < x { y } else { x }
```

Comment:
```suggestion
  yield if x > y { y } else { x }
```

It's irrelevant for `Int`, but in general I believe people expect `min[a, b]` to return `a` if `a == b`.